source/integrations/cloud-bot-accounts.rst

```diff
@@ -125,12 +125,12 @@ Do bot access tokens expire?
 
 No, but you can automate your integration to cycle its token `through the REST API <https://api.mattermost.com/#tag/users%2Fpaths%2F~1users~1%7Buser_id%7D~1tokens%2Fpost>`_.
 
-For more information about access tokens, see `the personal access tokens documentation <https://docs.mattermost.com/cloud/cloud-integrations/cloud-personal-access-tokens.html>`_.
+For more information about access tokens, see `the personal access tokens documentation <https://docs.mattermost.com/developer/integration-faq.html?highlight=cloud%20personal%20access>`_.
```

Comment:
@justinegeffen - Can you assist with identifying the correct URL needed for this link, please?

Reply:
I got the link. Updating with the correct one

Reply:
@cwarnermm